### ISSUE TITLE
Feature Request: support through bridges

### ISSUE THREAD

**Issue by mrvn** [alexrj/Slic3r#3096]
I'm (still, again :) printing a 40 teeth GT2 pulley with a hole for an M8 nut in the bottom (no, I can't print it upside down). A central shaft goes all the way through the pulley so the roof of the M8 nut hole still has a smaller hole in the middle. Like this:

          |      |
    +---+    +----+
    |                   |

One can print bridges many cm long without too much of a problem but due to the central shaft this is a 90° overhang instead of a bridge. And here comes my feature request:

Wouldn't it be cool if slicer could print the roof of the M8 nut hole at least partially closed, building bridges right across the central shaft as a kind of support. That way it wouldn't require any support build from the ground up and the central shaft can be easily drilled free. Or just push a pencil through to break the single layer of support bridges if they are sparse and fragile enough.

In other words I don't want the support build vertically or resting on top of other parts of the object but to have it rest horizontally on the object transforming overhangs into bridges.

**Comment by bcolsen** [comment]
I'm late to the party but, I thought a picture (and a work around) might help this proposal:  (I do appreciate ascii art but I can quite tell what's going on there...sorry)

![screenshot40](https://cloud.githubusercontent.com/assets/10513354/11003577/7f896808-8470-11e5-8957-43c12583b55e.png)

This a M3 nut handle I made with a pause and a drop in nut.  The next layer after the nut starts printing the center circle in mid air, not attached to the part.  This is clearly not going to work.  

It seems that this proposal(correct me if I'm wrong) would do the second case, which would solid bridge and then start the features of the layer.

Work around: I made these screen shots by changing my .stl by padding a solid 0.2 mm(or whatever your print height) layer below my feature.  The next layer then tries to do line thickening, which probably isn't ideal after a bridge, but it's better than printing in air.
